src/expressions/functions/builtInFunctions_string.ts

```diff
@@ -828,7 +850,13 @@ export default {
 			namespaceURI: FUNCTIONS_NAMESPACE_URI,
 			returnType: 'xs:integer*',
 		},
-
+		{
+			argumentTypes: ['xs:string?'],
+			callFunction: fnEncodeForUri,
+			localName: 'encode-for-uri',
+			namespaceURI: FUNCTIONS_NAMESPACE_URI,
+			returnType: 'xs:string?',
```

Comment:
Should that be just xs:string - I don't think this function ever returns an empty sequence?